.github/workflows/verify.yaml

```diff
@@ -29,4 +29,4 @@ jobs:
         run: deno lint
 
       - name: test
-        run: deno test
+        run: deno test -A
```

Comment:
nit: Do you think we should be more specific here with the permissions we allow?  In the context of a test runner inside CI, it's probably okay, just feels like we are removing a major benefit to using `deno`.

Reply:
It's a good point, and I agree that it's best to always constrain the permissions to the smallest possible.

I've added a task `test` that only enables the run permission on the `deno` executable. Also, I simplified the simulated daemon to just use a really long sleep. That way it doesn't need any special permission, and all we're testing is the `SIGINT` behavior.